howto/stream/exchange-oob-data.md

```diff
@@ -110,19 +138,23 @@ A trivial example to simulate the data transmission between an instance and a We
    anbox cloud <--  the received data
    ```
 
-### Anbox WebRTC data proxy
+This enables data exchange between a service running on the Anbox instance and the web client. However, it does not yet facilitate data exchange between an Android application running inside the Android container and the web client.
+
+
+### Data exchange between Android application and web client
 
-To build up the communication bridge between an Android application and the Anbox WebRTC platform, Anbox Cloud provides a system daemon named `anbox-webrtc-data-proxy`. This daemon is responsible for:
+To build up the communication bridge between an Android application and the web client, Anbox Cloud provides a system daemon named `anbox-webrtc-data-proxy`.  This daemon is responsible for:
 
+ * Registering a system service named `org.anbox.webrtc.IDataProxyService` in Android container
  * Accepting connection requests from an Android application
- * Connecting to one specific data channel via the Unix domain socket exposed by the Anbox WebRTC platform
+ * Connecting to a specific data channel via the Unix domain socket exposed by the Anbox runtime
  * Passing the connected socket as a file descriptor to the Android application
 
-The `anbox-webrtc-data-proxy` system daemon runs in the instance and registers an Android system service named `org.anbox.webrtc.IDataProxyService`. This service allows Android developers to easily make use of binder interprocess communication (IPC) for data communication between an Android application and the Anbox WebRTC platform through a file descriptor.
+This allows Android developers to easily make use of binder interprocess communication (IPC) for data communication between an Android application and the Anbox runtime through a file descriptor, enabling further data exchange with the web client.
```

Comment:
How does binder come into the picture? What is it? We haven't mentioned it abive, just "Registering a system service named ..."

Reply:
Let's just mention the Android system service for clarity.